.gitmodules

```diff
@@ -1,6 +1,6 @@
-[submodule "osu-framework"]
-	path = osu-framework
-	url = https://github.com/ppy/osu-framework
-[submodule "osu-resources"]
-	path = osu-resources
-	url = https://github.com/ppy/osu-resources
+[submodule "osu-framework"]
+	path = osu-framework
+url=https://github.com/Memorix101/osu-framework
+[submodule "osu-resources"]
+	path = osu-resources
+	url = https://github.com/ppy/osu-resources
```

Comment:
.gitmodules should not be commit, and we should probably/definitely add a .gitignore entry for this.